catalyst/runners/config.py

```diff
@@ -419,6 +420,54 @@ def get_callbacks(self, stage: str) -> "OrderedDict[str, Callback]":
         return callbacks
 
 
+class SelfSupervisedConfigRunner(ISelfSupervisedRunner, ConfigRunner):
+    """ConfigRunner for contrastive tasks
+
+    Args:
+        config: dictionary with parameters
+        input_key: key in ``runner.batch`` dict mapping for model input
+        output_key: key for ``runner.batch`` to store model output
+        target_key: key in ``runner.batch`` dict mapping for target
+        loss_key: key for ``runner.batch_metrics`` to store criterion loss output
+        augemention_prefix: key for ``runner.batch`` to sample augumentions
+        projection_prefix: key for ``runner.batch`` to store model projection
+        embedding_prefix: key for `runner.batch`` to store model embeddings
+
+    .. note::
+        Please follow the `minimal examples`_ sections for use cases.
+
+        .. _`minimal examples`: https://github.com/catalyst-team/catalyst#minimal-examples
+
+    Examples:
+
+    .. code-block:: python
+
+        dataset = SomeDataset()
+    """
+
+    def __init__(
+        self,
+        config: Dict = None,
+        input_key: str = "features",
+        target_key: str = "target",
+        loss_key: str = "loss",
+        augemention_prefix: str = "aug",
```

Comment:
```suggestion
        augemention_prefix: str = "augment",
```
? I am not sure if the `aug` is a good name practice

Reply:
it's a proposal

Reply:
👀